set_version.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Could you please move this script to the `scripts/` directory?

Thanks for adding it! Quite useful.